deepchecks/tabular/_shared_docs.py

```diff
@@ -33,18 +33,18 @@
 """.strip('\n')
 
 _shared_docstrings['feature_aggregation_method_argument'] = """
-argument for the reduce_output functionality, decides how to aggregate the drift scores for a
+argument for the reduce_output functionality, decides how to aggregate the per-feature scores into a
 collective score. The collective score value is between 0 and 1 for all methods other than l2_combination.
 Possible values are:
-'l2_weighted': L2 norm over the combination of drift scores and feature importance, minus the
-L2 norm of feature importance alone, specifically, ||FI + DRIFT|| - ||FI||. This method returns a
+'l2_weighted': L2 norm over the combination of per-feature scores and feature importance, minus the
+L2 norm of feature importance alone, specifically, ||FI + SCORE|| - ||FI||. This method returns a
```

Comment:
I think that the "SCORE" here is unclear. If this can't be solved easily then maybe consider splitting the doc into 2 separate paragraphs.

Reply:
What two paragraphs would you suggest? This section is preamble + description of the individual options.
Will rethink SCORE though

Reply:
Good enough, I think. See my second suggestion. Anyway approved